examples/md-flexible/src/Simulation.cpp

```diff
@@ -109,10 +119,30 @@ Simulation::Simulation(const MDFlexConfig &configuration,
   _autoPasContainer = std::make_shared<autopas::AutoPas<ParticleType>>(*_outputStream);
   _autoPasContainer->setAllowedCellSizeFactors(*_configuration.cellSizeFactors.value);
   _autoPasContainer->setAllowedContainers(_configuration.containerOptions.value);
-  _autoPasContainer->setAllowedDataLayouts(_configuration.dataLayoutOptions.value);
-  _autoPasContainer->setAllowedNewton3Options(_configuration.newton3Options.value);
-  _autoPasContainer->setAllowedTraversals(_configuration.traversalOptions.value);
+
+  if (_configuration.getInteractionTypes().size() == 0) {
+    _configuration.functorOption.value = MDFlexConfig::FunctorOption::lj12_6;
+    _configuration.addInteractionType(autopas::InteractionTypeOption::pairwise);
+    std::cout << "WARNING: No functor was specified. Using the default Lennard-Jones AutoVec functor." << std::endl;
+  }
```

Comment:
This changes the default behavior of md-flex. Previously, the default functor was the AVX or SVE intrinsic one, depending on which instruction set was available. This was also very important because students would always forget to use the AVX functor even when told to use it ;)
Please restore this behavior.

This was implemented here: https://github.com/AutoPas/AutoPas/blob/1f6719ab7db7ce4d9ee5aac81885f55d4f8cf200/examples/md-flexible/src/configuration/MDFlexConfig.h#L440-L452

Reply:
Ah okay, I didn't deem this important :D 
It's reimplemented, but in `Simulation.h`, see [f1acdde](https://github.com/AutoPas/AutoPas/pull/751/commits/f1acdde45f7983c7f96b79bf1b51e65298fcf122)